cmd/webpackager/flag_packager.go

```diff
@@ -48,6 +49,9 @@ var (
 	flagCertURL      = flag.String("cert_url", "", `Certficiate chain URL. (required)`)
 	flagPrivateKey   = flag.String("private_key", "", `Private key PEM file. (required)`)
 
+	// Processor
+	flagSizeLimit = flag.String("size_limit", "4194304", `Maximum size of resources allowed for signed exchanges, or "none" to set no limit.`)
```

Comment:
optional: Maximum size of resources in bytes

Reply:
optional: re-use noSizeLimitString here, instead of writing "none" literally.

Reply:
Thank you!

Reply:
Updated the help description. Glad to know it's something really wanted.